hugo/layouts/partials/site/head.html

```diff
@@ -1,5 +1,11 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="description" content="
+{{ with .Description | $.RenderString | plainify }}{{ . }}
```

Comment:
you do need dashes for beginning and end tags now {{-   and -}} because now new lines inside the code also end up in the description tag